backend/run_debug.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Quick script to run the backend with virtual environment activated
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)